cirq/ops/linear_combinations_test.py

```diff
@@ -562,3 +562,20 @@ def test_linear_combination_of_operations_has_correct_pauli_expansion(
 ))
 def test_operation_expressions(expression, expected_result):
     assert_linear_combinations_are_equal(expression, expected_result)
+
+
+def test_pauli_sum_from_adding_strings():
+    q = cirq.LineQubit.range(2)
+    pstr1 = cirq.X(q[0]) * cirq.X(q[1])
+    pstr2 = cirq.Y(q[0]) * cirq.Y(q[1])
+    psum = pstr1 + pstr2
```

Comment:
Wait, `__add__` is inferred from `__iadd__`? Could you verify that you didn't mutate `pstr1` or `pstr2` by performing this addition?

Reply:
this is from PauliString.__add__

Reply:
:man_facepalming: